mlfinlab/backtest_statistics/statistics.py

```diff
@@ -0,0 +1,233 @@
+"""
+Implements the book Chapter 14 on Backtest Statistics
```

Comment:
Better docstring please.

Reply:
Will make same description as in the ___init___.py

Reply:
No, the init file is for the high level folder which can have many files. This file is an individual and we should have a docstring that describes this file.